pkg/migration-tool/planner/planner.go

```diff
@@ -161,19 +162,19 @@ func (p *Plan) createBlock(mint, maxt int64) (reference *Block, err error) {
 		return nil, fmt.Errorf("create-block: %w", err)
 	}
 	id := p.blockCounts.Add(1)
-	timeRangeInMinutes := (maxt - mint) / time.Minute.Milliseconds()
+	timeRangeInMinutes := (maxt - mint) / minute
 	percent := float64(maxt-p.config.Mint) * 100 / float64(p.config.Maxt-p.config.Mint)
 	if percent > 100 {
 		percent = 100
 	}
-	baseDescription := fmt.Sprintf("progress: %.3f%% | block-%d time-range: %d mins | mint: %d | maxt: %d", percent, id, timeRangeInMinutes, mint, maxt)
+	baseDescription := fmt.Sprintf("progress: %.3f%% | block-%d time-range: %d mins | mint: %d | maxt: %d", percent, id, timeRangeInMinutes, mint/second, maxt/second)
 	reference = &Block{
 		id:                    id,
 		pbarDescriptionPrefix: baseDescription,
 		pbar: progressbar.NewOptions(
-			6,
+			4,
```

Comment:
Sorry, the 4 represents the number of steps taken by a progressbar to get to the full. The 2 steps decreased here is because this progress bar is now independent of the reader's functionalities (or steps) (handled by bytes bar in `client.go`), thus decreasing its journey to full.

Reply:
create a named const so it's self documenting